code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -112,13 +112,15 @@
 
 /mob/living/silicon/bullet_act(obj/item/projectile/Proj, def_zone)
 	SEND_SIGNAL(src, COMSIG_ATOM_BULLET_ACT, Proj, def_zone)
-	if((Proj.damage_type == BRUTE || Proj.damage_type == BURN))
+	if(Proj.damage_type == BRUTE)
 		adjustBruteLoss(Proj.damage)
-		if(prob(Proj.damage*1.5))
-			for(var/mob/living/M in buckled_mobs)
-				M.visible_message("<span class='boldwarning'>[M] is knocked off of [src]!</span>")
-				unbuckle_mob(M)
-				M.Knockdown(40)
+	if(Proj.damage_type == BURN)
+		adjustFireLoss(Proj.damage)
```

Comment:
Why do they even snowflake this crap? This should probably just get removed entirely in favor of borgs taking damage from the bullet itself, let the bullet handle it.

Reply:
someone else can refactor it i aint a coder